woocommerce/wc-admin
# Correct the Klarna slug

Fixes #6435

This fixes the Klarna slug in the `Klarna` component (`klarna.js`) to use underscores instead of hyphens. It also makes the method query throw a descriptive error if the method isn't found, rather than swallowing it in a non-descriptive null reference error.

### Detailed test instructions:

1. Set up a new store with Klarna available as a payment processor
2. Go to the "Choose payment methods" task item
3. Set up Klarna. The plugin will install.
4. Click Continue. It should take you back to the payment methods page - previously it wasn't doing anything but a console error was displayed.
